.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - '**.md' #Do not need to run CI for markdown changes.
+  pull_request:
+    branches: [main]
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build-test:
+    strategy:
+      matrix:
+        variations: [
+          # {os: ubuntu-latest, node: latest},
```

Comment:
Disabled due to this bug: https://github.com/npm/cli/issues/7657

Running on the latest LTS is already a better target, and we can enable this again if that bug is resolved. Or we use a combination of dependencies that doesn't trigger it.